src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,12 @@
+{"properties": [
```

Comment:
What does this do?  Why do we need it?  What problem is it fixing?

Reply:
The file got corrupted.